source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+        ...26%
+   Extracting storage-utils-dev pack for version 18770
+        ...31%
+   Extracting os-core-update-dev pack for version 18760
+        ...36%
+   Extracting network-basic-dev pack for version 18760
+        ...42%
+   Extracting mixer pack for version 18790
+        ...47%
+   Extracting os-installer pack for version 18800
+        ...52%
+   Extracting mail-utils-dev pack for version 18760
+        ...57%
+   Extracting koji pack for version 18800
+        ...63%
+   Extracting go-basic pack for version 18800
+        ...68%
+   Extracting dev-utils-dev pack for version 18820
+        ...73%
+   Extracting python-basic-dev pack for version 18750
+        ...78%
+   Extracting perl-basic-dev pack for version 18610
+        ...84%
+   Extracting c-basic pack for version 18800
+        ...89%
+   Extracting os-core-dev pack for version 18800
+        ...94%
+   Extracting web-server-basic pack for version 18680
+        ...100%
+   Installing bundle(s) files...
+        ...100%
+   Calling post-update helper scripts.
+   Possible filedescriptor leak : 8 (socket:[30833])
+   Bundle(s) installation done.
+
+.. note::
+
+   This script uses :command:`sudo` to run commands and you may be prompted to
+   enter your user password at any time while the script is executing. If this
+   occurs, enter your user password to execute the :command:`sudo` command.
+
+   .. code-block:: console
+
+      Password:
+
+   You may also see an informational message about setting the
+   :guilabel:`https_proxy` environment variable if this variable isn't set.
+
+Once the :command:`swupd` command is complete, the script begins processing
+the requirements to install and implement the telemetrics server and enable
+it. The script output will look similar to this:
+
+.. code-block:: console
+   
+   Collecting uwsgitop
+     Downloading uwsgitop-0.10.tar.gz
+   Requirement already satisfied: simplejson in /usr/lib/python3.6/site-packages (from uwsgitop)
+   Collecting argparse (from uwsgitop)
+     Downloading argparse-1.4.0-py2.py3-none-any.whl
+   Building wheels for collected packages: uwsgitop
+     Running setup.py bdist_wheel for uwsgitop ... done
+     Stored in directory: /root/.cache/pip/wheels/8a/99/e9/accc80bcaa989218da65daaae4205dc4f6288d3551655aa638
+   Successfully built uwsgitop
+   Installing collected packages: argparse, uwsgitop
+   Successfully installed argparse-1.4.0 uwsgitop-0.10
+   mkdir: created directory '/var/www'
+   mkdir: created directory '/var/www/telemetry'
+   Already using interpreter /usr/bin/python3
+   Using base prefix '/usr'
+   New python executable in /var/www/telemetry/venv/bin/python3
+   Also creating executable in /var/www/telemetry/venv/bin/python
+   Installing setuptools, pip, wheel...done.
+   Collecting alembic==0.9.5 (from -r /tmp/requirements.txt.KDI3uU (line 1))
+     Downloading alembic-0.9.5.tar.gz (990kB)
+       100% |████████████████████████████████| 993kB 2.1MB/s
```

Comment:
Will the web site display these solid block characters correctly? It looks like they are U+2588 (full block) characters. It might be worth checking to see if fonts tend to include it, since I'm not familiar with it. Otherwise, I think replacing these characters with asterisks or equals signs would be acceptable.

Reply:
Since the content is placed within a code-block it will be rendered as UTF-8 code. I'll test to make sure but the blocks should render properly.